packages/react-native-reanimated/android/src/main/worklets/CMakeLists.txt

```diff
@@ -0,0 +1,143 @@
+cmake_minimum_required(VERSION 3.8)
+
+set(LOCAL_PACKAGE_NAME "worklets")
+
+file(GLOB_RECURSE WORKLET_COMMON_SOURCES CONFIGURE_DEPENDS "${WORKLET_COMMON_DIR}/*.cpp")
+
+# Consume shared libraries and headers from prefabs
+find_package(fbjni REQUIRED CONFIG)
+find_package(ReactAndroid REQUIRED CONFIG)
+
+if(${JS_RUNTIME} STREQUAL "hermes")
+    find_package(hermes-engine REQUIRED CONFIG)
+endif()
+
+add_library(
+    ${LOCAL_PACKAGE_NAME}
+    SHARED
+    ${WORKLET_COMMON_SOURCES}
+    ${INCLUDE_JSI_CPP}
+    ${INCLUDE_JSIDYNAMIC_CPP}
+    "${ANDROID_SRC_DIR}/main/cpp/AndroidLogger.cpp"
+)
+
+# includes
+target_include_directories(
+    ${LOCAL_PACKAGE_NAME}
+    PUBLIC
+    "${WORKLET_COMMON_DIR}/hidden_headers"
+    "${WORKLET_COMMON_DIR}/Registries"
+    "${WORKLET_COMMON_DIR}/SharedItems"
+    "${WORKLET_COMMON_DIR}/Tools"
+    "${WORKLET_COMMON_DIR}/WorkletRuntime"
+)
+
+target_include_directories(
+    ${LOCAL_PACKAGE_NAME}
+    PRIVATE
+    "${ANDROID_SRC_DIR}/main/cpp"
+)
+
+target_include_directories(
+    ${LOCAL_PACKAGE_NAME}
+    PRIVATE
+    "${REANIMATED_COMMON_DIR}/NativeModules"
+    "${REANIMATED_COMMON_DIR}/AnimatedSensor"
+    "${REANIMATED_COMMON_DIR}/Fabric"
+    "${REANIMATED_COMMON_DIR}/LayoutAnimations"
+    "${REANIMATED_COMMON_DIR}/Tools"
+)
```

Comment:
The plan was that in `worklets` we don't include anything from `reanimated`.

Reply:
Yes, but it's impossible right now. Further refactors will come with follow-up PRs.